### ISSUE TITLE
PDFs are missing print and download buttons

### ISSUE THREAD

**Issue by LaurenWags** [brave/brave-browser#42970]
### Description

PDFs no longer display the print and download buttons. This broke between `1.75.67` and `1.75.68`, maybe due to https://github.com/brave/brave-core/pull/26676.

Found by @emerick 😄 

### Steps to reproduce

1. Using version `1.75.68` or higher, navigate to a PDF such as https://file-examples.com/storage/feb3ceaed86762e98a0a444/2017/10/file-sample_150kB.pdf
2. Look for print and download buttons (should be next to the 3 dots menu)
3. They are not visible, but if you hover over where they should be the areas highlight

### Actual result

Example | Example
--- | ---
![Image](https://github.com/user-attachments/assets/8c0132f9-0198-4d26-be51-b84318d3b003) | ![Image](https://github.com/user-attachments/assets/9a7141cd-a701-440c-98ff-65304b768d82)

### Expected result

![Image](https://github.com/user-attachments/assets/33548238-1793-4ab7-a16a-84731da09639)

### Reproduces how often

Easily reproduced

### Brave version (brave://version info)

Nightly `1.75.68 Chromium: 132.0.6834.15` and higher.

Does not reproduce in Beta, `1.74.29 Chromium: 132.0.6834.46`

### Channel information

- [ ] release (stable)
- [ ] beta
- [x] nightly

### Reproducibility

- [ ] with Brave Shields disabled
- [ ] with Brave Rewards disabled
- [ ] in the latest version of Chrome

### Miscellaneous information

cc @rebron @fallaciousreasoning @brave/qa-team

**Comment by fallaciousreasoning** [comment]
I don't think that PR should have affected this but I could be wrong 😆 - I'll take a look when I get a chance :)

**Comment by MadhaviSeelam** [comment]
Still seeing the same issue.  The issue is still open and not fixed in Brave | 1.75.164 Chromium: 132.0.6834.83 (Official Build) beta (64-bit). 

cc: @rebron @fallaciousreasoning 

example | example 
----- | -----
![Image](https://github.com/user-attachments/assets/2b2f28eb-a8d6-4e51-8efe-6658ddd5f334)|![Image](https://github.com/user-attachments/assets/08769394-e2af-4e29-b2ca-ca371fbb6da8)

**Comment by fallaciousreasoning** [comment]
This is super weird - it doesn't seem to show up in upstream Chrome either

**Comment by rebron** [comment]
Looks good in the following version. I see the print and download buttons with both buttons functioning properly.

Brave | 1.76.30 Chromium: 132.0.6834.83 (Official Build) nightly (arm64)
-- | --
Revision | 5dd7daea3c59dfe05f91cf0c6768ec595927c85f
OS | macOS Version 15.2 (Build 24C101)
JavaScript | V8 13.2.152.27
User Agent | Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/132.0.0.0 Safari/537.36

<img width="1235" alt="Image" src="https://github.com/user-attachments/assets/bb21f5e0-254b-45a3-b128-2e26306e9c8a" />

**Comment by rebron** [comment]
Good on Windows as well. Both buttons appear and are functional.

Brave	1.76.30 Chromium: 132.0.6834.83 (Official Build) nightly (64-bit) 
Revision	5dd7daea3c59dfe05f91cf0c6768ec595927c85f
OS	Windows 11 Version 24H2 (Build 26100.2605)
JavaScript	V8 13.2.152.27
User Agent	Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/132.0.0.0 Safari/537.36

<img width="605" alt="Image" src="https://github.com/user-attachments/assets/fe776d86-72c2-4f0f-b5e2-37ea1b1a69da" />

**Comment by kjozwiak** [comment]
The above requires `1.75.166` or higher for `1.75.x` verification 👍

**Comment by MadhaviSeelam** [comment]
Verification `PASSED` using
```
Brave | 1.75.173 Chromium: 133.0.6943.39 (Official Build) (64-bit)
-- | --
Revision | 21435b152a0c070861f182a0b6782222a8efae37
OS | Windows 11 Version 24H2 (Build 26100.2894)
```
Verified the issue using the STR from https://github.com/brave/brave-browser/issues/42970#issue-2748760761

Confirmed print and download buttons are shown and working as expected 

example | example | example | example 
------ | ----- | ---- | ----
<img width="957" alt="Image" src="https://github.com/user-attachments/assets/b14575ed-6385-4943-add5-7ecaf79e8041" />|![Image](https://github.com/user-attachments/assets/286b7af9-650e-4c4c-9e72-67500724de42)|![Image](https://github.com/user-attachments/assets/bd0b7acc-6e25-4c64-87e8-a82a4db7750c)|![Image](https://github.com/user-attachments/assets/e3952f95-5bb3-4efa-afda-97f87c5d6e7a)
